tasks-by-users/Tauqeer-Ahmed-99/07-JS-ToDo-Classes-and-OOP/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <script src="script.js" defer></script>
+    <title>Document</title>
+  </head>
+  <body>
+    <div id="container">
+      <header class="heading">To Do Web Application</header>
+      <div class="columns">
+        <section class="col active">
+          <div class="header"><h1 class="head">Tasks to Complete:</h1></div>
+          <!-- <div class="header"><h1 class="head">Tasks Completed:</h1></div> -->
```

Comment:
remove unnecessary comments

Reply:
removed commented code please approve @faran4engg ..
& sorry for late response...